src/hooks/usePersistedFilter.ts

```diff
@@ -25,8 +25,8 @@ export function usePersistedFilter<F>({ storageKey, filterDefs }: UsePersistedFi
     () =>
       Object.fromEntries(
         safeEntries(filterDefs)
-          .filter(([, def]) => def.defaultValue !== undefined)
-          .map(([key, def]) => [key, (def as any).defaultValue]),
+          .filter(([key, def]) => def(key as string).defaultValue !== undefined)
+          .map(([key, def]) => [key, def(key as string).defaultValue]),
```

Comment:
Was this broken?

Reply:
No, but `def.defaultValue` now has to be `def(key).defaultValue` b/c of the "`singleFilter` returns a factory that takes the filter's key" approach.